src/main/scala/mesosphere/marathon/core/storage/StorageConfig.scala

```diff
@@ -29,10 +29,11 @@ import scala.collection.immutable.Seq
 import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.{ Duration, _ }
 import scala.reflect.ClassTag
+// scalastyle:on
 
 sealed trait StorageConfig
```

Comment:
(1) Sounds good. I did move to StorageConf.
(2) Hmm. We could do that, the repository package has generic repositories too and most of the marathon repositories are just named editions of them.